diff --git a/node_modules/ocad2geojson/src/ocad-to-svg.js b/node_modules/ocad2geojson/src/ocad-to-svg.js
index 1101f1b..0f0ed42 100644
--- a/node_modules/ocad2geojson/src/ocad-to-svg.js
+++ b/node_modules/ocad2geojson/src/ocad-to-svg.js
@@ -185,6 +185,7 @@ const createSvgNode = (document, n) => {
       Object.keys(n.attrs).forEach(attrName =>
         node.setAttribute(attrName, n.attrs[attrName])
       )
+    if (n.order !== undefined) node.setAttribute('data-order', String(n.order))
   } else {
     node = document.createTextNode(n.text)
   }
@@ -389,16 +390,6 @@ const objectToSvg = (options, symbols, object) => {
             ],
           ]
             .map(([offset, width, color]) => {
-              const a = object.coordinates.map(c => ({
-                type: 'circle',
-                attrs: {
-                  cx: c[0].toString(),
-                  cy: (-c[1]).toString(),
-                  r: '3',
-                  fill: 'red',
-                },
-              }))
-
               return offsetLineCoordinates(
                 object.coordinates,
                 offset,
@@ -414,7 +405,6 @@ const objectToSvg = (options, symbols, object) => {
                     options.closePath
                   )
                 })
-                .concat(a)
             })
             .flat()
         )
